docs/docsite/rst/plugins/process.rst

```diff
@@ -0,0 +1,50 @@
+.. _process_plugins:
+
+Process Model Plugins
+=====================
+
+.. contents::
+   :local:
+   :depth: 2
+
+Process plugins control the way Ansible creates workers to run tasks on hosts. Currently, the specified process model is global and cannot be changed once Ansible starts.
+
+By default, Ansible ships with two process models: :ref:`forking<forking_process>` and :ref:`threading<threading_process>`.
```

Comment:
Some information here about why someone might want to change their process model - not necessarily pointers on which one to choose, but more general considerations. For example (and I'm making this up, but it's a semi-educated guess): "Selecting the right process plugin can improve Ansible's performance in your environment. You should match the process model to your available resources." Or maybe: "Process models have security implications. You should select a process plugin with care to ensure secure operations." Whatever the specifics, an indication of why the user might care will improve this page.